src/core_modules/capture-core/components/Pages/EnrollmentAddEvent/EnrollmentAddEventPageDefault/EnrollmentAddEventPageDefault.container.js

```diff
@@ -52,13 +52,15 @@ export const EnrollmentAddEventPageDefault = ({
     const selectedProgramStage = [...program.stages.values()].find(item => item.id === stageId);
     const outputEffects = useWidgetDataFromStore(widgetReducerName);
     const hideWidgets = useHideWidgetByRuleLocations(program.programRules);
+    // $FlowFixMe
+    const trackedEntityName = program?.trackedEntityType?.name;
```

Comment:
This seems like a dirty hack from my side. Any ideas on any better solutions? Enrollment does no longer contain the TET.